src/util/color/colorpalette.cpp

```diff
@@ -0,0 +1,12 @@
+#include "colorpalette.h"
+
+const ColorPalette ColorPalette::mixxxPalette =
+        ColorPalette(QList<mixxx::RgbColor>{
+                mixxx::RgbColor(0xC50A08),
```

Comment:
Could we name these predefined colors and define them as individual constants? Would improve readability.